### DISCUSSION TITLE
Multi disk keywords

### DISCUSSION THREAD

**Discussion by rortonuk** [beetbox/beets#5446] [Q&A]
having a bit of trouble with beets recognising multi disks with a certain directory name. I have some multi disks albums in directories called Digital Media 1 and Digital media 2 (not a disk per say I know but still separated)

is there a way to get beets to recognise this naming structure as a multi disks. I guess I could go through and manually rename these directories to cd1 cd1 etc but would prefer not to if possible?

**Comment by jackwilsdon** [comment]
beets only automatically recognises multi-disk albums with "disc", "disk" or "cd" in the name: https://github.com/beetbox/beets/blob/04ee04150ac6ce41393c4e8e0d04030f3c5e2596/beets/importer.py#L1748-L1748

Your best option might be to use the `--flat` flag, which will treat all files in a directory (recursively) as being part of the same album: https://beets.readthedocs.io/en/stable/reference/cli.html#import

I'm not too sure how this handles multiple directories passed on the command line, so it may need a bit of experimentation.
